src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2125,7 +2091,7 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
             throw new LogicException('Messenger support cannot be enabled as the Messenger component is not installed. Try running "composer require symfony/messenger".');
         }
 
-        if (!$this->hasConsole() || !class_exists(StatsCommand::class)) {
```

Comment:
Class was added to symfony/messenger with #46571 (6.2), FrameworkBundle requires 6.3+.